kotlin/android/README.md

```diff
@@ -1,24 +1,40 @@
 # Firezone Android client
 
-## Prerequisites for developing locally
+This README contains instructions for building and testing the Android client
+locally.
```

Comment:
I have had good experiences with following a one-sentence-per-line pattern when it comes to documentation stored in Git because Git's diffing algorithm works on a line-by-line basis.

Additionally, it makes it quite efficient to collaborate on these docs via GitHub's suggestion feature in PRs.

Don't feel strongly though if you want to retain wrapping at a certain character count.

Reply:
I agree think this makes sense, but it would be great if `prettier` or `remark` supported this -- are you wrapping these manually? If we can't format / lint it then I'd prefer to just stick at a character count wrap so that when working on the docs in `website/` everyone's editor behaves.

See https://prettier.io/docs/en/options.html#prose-wrap and https://github.com/firezone/firezone/blob/main/.prettierrc.json